app/javascript/components/patient/Patient.js

```diff
@@ -68,665 +91,622 @@ class Patient extends React.Component {
     return (
       <React.Fragment>
         <Row id="monitoree-details-header">
-          <Col className="mt-1">
-            <div className="h3">
+          <Col sm={12}>
+            <h3>
               <span className="pr-2">
                 {`${this.props.details.first_name ? this.props.details.first_name : ''}${
                   this.props.details.middle_name ? ' ' + this.props.details.middle_name : ''
                 }${this.props.details.last_name ? ' ' + this.props.details.last_name : ''}`}
               </span>
               {this.props?.dependents && this.props?.dependents?.length > 0 && <BadgeHOH patientId={String(this.props.details.id)} location={'right'} />}
-            </div>
+            </h3>
+            <div className="clearfix"></div>
```

Comment:
This do anything? I can't find the class `clearfix` and it's rendering as nothing in the DOM

Reply:
this must have been a carry over of some sort.  I removed it and don't see any weird behavior